app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3 and command_list[0] == "mv":
+        old_name, new_name = command_list[1:]
+        if "/" not in new_name:
+            pass
+        else:
+            new_name_split = new_name.split("/")
+            path = new_name_split[0]
+            for index in range(1, len(new_name_split)):
+                if os.path.exists(path) is False:
+                    os.makedirs(path)
```

Comment:
The `os.makedirs(path)` function should be called with the full path, not just the current segment. Ensure the path is correctly updated before this call.